tests/gh-r.zunit

```diff
@@ -145,7 +145,7 @@
   $dive --version; assert $state equals 0
 }
 @test 'docker-buildx' { # A monitor of resources
-  run zinit for as'completions' atclone'buildx* completion zsh > _buildx' lbin'!buildx-* -> buildx' @docker/buildx
+  run zinit for as'completions' atclone'./buildx* completion zsh > _buildx' lbin'!buildx-* -> buildx' @docker/buildx
```

Comment:
Un-related, please remove

Reply:
It fails the unit tests without as before the atclone ice wasn't run in the tests. If I remove this the CI run will fail (again)

(same for two more "fix: *" commits)

Reply:
Example session: 

```
[22:44:37] [1] ✗  zinit for as'completions' atclone'buildx* completion zsh > _buildx' lbin'!buildx-* -> buildx' @docker/buildx

Downloading docker/buildx…
Cloning into '/Users/jankatins/.zinit/plugins/docker---buildx'...
⠧ ███████████ OBJ: 100, PACK: 21227/21227, COMPR: 100%, REC: 100%, RES: 100%
No files for compilation found.
Warning: ∞zinit-compile-plugin-hook hook returned with 1
(eval):1: no matches found: buildx*
Warning: ∞zinit-atclone-hook hook returned with 1
linkbin annex: The lbin ice (`buildx-* -> buildx')didn't match any files
```